robot/basestation/templates/elements/navigation/Navigation.html

```diff
@@ -38,7 +38,7 @@ <h5>dist (m): <span id="distance-to-rover">-</span></h3>
   <br>
   <button type="button" id="send-goal-pos-btn" class="btn btn-primary">Send</button>
 </div>
-<div style="display: none" id="goal-unchanging">
+<div class="navigation" style="display: none" id="goal-unchanging">
```

Comment:
CS

Reply:
I'm going to remove the inline CSS for the other cases but I think this one should be kept since it's an exception

Reply:
Fixed